docs/layout-doc.md

```diff
@@ -4,59 +4,62 @@ The Layout layer is where thing get a bit more concrete, it might/should be extr
 
 Every layout in the library has to implement the `ELayout` interface which forces them to implement the following:
 
-`val childLayouts: List<ELayout>` : List of child layout. Can be empty and/or mutable
+`val childLayouts: List<ELayout>` : List of child layouts. Can be empty and/or mutable
+
+//TODO: what does being given meant/size you actually need?
 
 `fun size(toFit: ESize): ESize`
-It's the equivalent of Android's `onMeasure`, your being given and you need to return the size you actually need.
+It's the equivalent of Android's `onMeasure()`, you're being given and you need to return the size you actually need.
 
-In the Android framework there is this concept of **Mode** which can be AT_MOST,EEACTLY or UNSPECIFIED. In this library this concept is abstracted thus not required.
+In the Android framework, there is this concept of **Mode** which can be AT_MOST, EXACTLY or UNSPECIFIED. In this library, this concept is abstracted and thus not required.
 
 `fun arrange(frame: ERect)`
 This function is like the `onLayout` function of Android, you're given a frame and the layout has to arrange itself into it
 
+//TODO: It's TBC so I guess it's not super important but you could have particular layouts ignore the frame and size?
 
-**TBC**: `arrange()` and `size()` don't have to obligation to respect the parameter being passed to them, but should most of the time.
+**TBC**: `arrange()` and `size()` don't have an obligation to respect the parameter being passed to them, but should most of the time.
```

Comment:
It's TBC so I guess it's not super important but you could have particular layouts ignore the frame and size?

Reply:
I've prototyped example where the parameter given in size() is ignored, but not with arrange()'s frame